src/tests/tracing/eventcounter/runtimecounters.cs

```diff
@@ -16,37 +16,41 @@ namespace RuntimeEventCounterTests
 {
     public class RuntimeCounterListener : EventListener
     {
-        public RuntimeCounterListener()
-        {
-            observedRuntimeCounters = new Dictionary<string, bool>() {
-                { "cpu-usage" , false },
-                { "working-set", false },
-                { "gc-heap-size", false },
-                { "gen-0-gc-count", false },
-                { "gen-1-gc-count", false },
-                { "gen-2-gc-count", false },
-                { "threadpool-thread-count", false },
-                { "monitor-lock-contention-count", false },
-                { "threadpool-queue-length", false },
-                { "threadpool-completed-items-count", false },
-                { "alloc-rate", false },
-                { "active-timer-count", false },
-                { "gc-fragmentation", false },
-                { "gc-committed", false },
-                { "exception-count", false },
-                { "time-in-gc", false },
-                { "gen-0-size", false },
-                { "gen-1-size", false },
-                { "gen-2-size", false },
-                { "loh-size", false },
-                { "poh-size", false },
-                { "assembly-count", false },
-                { "il-bytes-jitted", false },
-                { "methods-jitted-count", false },
-                { "time-in-jit", false }
-            };
-        }
-        private Dictionary<string, bool> observedRuntimeCounters;
+        private readonly Dictionary<string, bool> observedEvents = new Dictionary<string, bool>() {
+            { "cpu-usage" , false },
+            { "working-set", false },
+            { "gc-heap-size", false },
+            { "gen-0-gc-count", false },
+            { "gen-1-gc-count", false },
+            { "gen-2-gc-count", false },
+            { "threadpool-thread-count", false },
+            { "monitor-lock-contention-count", false },
+            { "threadpool-queue-length", false },
+            { "threadpool-completed-items-count", false },
+            { "alloc-rate", false },
+            { "active-timer-count", false },
+            { "gc-fragmentation", false },
+            { "gc-committed", false },
+            { "exception-count", false },
+            { "time-in-gc", false },
+            { "gen-0-size", false },
+            { "gen-1-size", false },
+            { "gen-2-size", false },
+            { "loh-size", false },
+            { "poh-size", false },
+            { "assembly-count", false },
+            { "il-bytes-jitted", false },
+            { "methods-jitted-count", false },
+            { "time-in-jit", false },
+            // These are AppContext switches, not counters
+            { "appContextSwitch", false },
+            { "appContextBoolAsStringData", false },
```

Comment:
You can prevent these from showing up by excluding the keyword you added from the call to `EnableEvents` down on L61.  Something like

```csharp
EnableEvents(source, EventLevel.Informational, (EventKeywords)((-1) & ~(int)(RuntimeEventSource.Keywords.AppContext)), refreshInterval);
```

Reply:
Is this something I would need only if we split this into a different test? Right now I want to see all events